pages/studentPortal/quizcomponent/FirstProgrammingLanguageQuiz.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from "react";
+import LangResults from "../../../components/resources/quizzes/langQuiz/LangResults";
+import BluePrint from "../../../components/resources/quizzes/shared/BluePrint";
+import SkillSelections from "../../../components/resources/quizzes/shared/SkillSelections";
+import StartQuiz from "../../../components/resources/quizzes/shared/StartQuiz";
+export enum Stage {
+  START,
+  LEARNCODING,
+  WORKFIELDS,
+  BUILDTARGET,
+  BLUEPRINT,
+  RESULTS,
+}
+
+type QuizSelection = {
+  name: string;
+  result?: string;
+  weight?: number;
+};
+
+const FirstProgrammingLanguageQuiz = () => {
+  const quiz: QuizSelection[][] = [
+    [
+      { name: "Learn an in-demand skill" },
```

Comment:
Question for @LuckyHariharan as well

Are the quiz questions going to live in hasura? If so we should eventually fetch this from the db. If not this should move server side

Reply:
lets move to server side

Reply:
take a look at `pages/api/studentPortal/units.ts` I think it's easy enough to move these quiz question objects somewhere into the api folder. It'll clean up this component, and separate the UI from the logic a bit more. 

I think it's a small enough change to do in this PR